curriculum/challenges/english/25-front-end-development/lab-leap-year-calculator/66c06fad3475cd92421b9ac2.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 66c06fad3475cd92421b9ac2
+title: Build a Leap Year Calculator
+challengeType: 14
+dashedName: build-a-leap-year-calculator
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. Declare a variable `year` that stores the value of the year to check.
+
+1. Use an `if/ else` statement to check if the year is a leap year.
```

Comment:
technically this could also be solved with a ternary which they will have learned about before this lab.
So I vote we update the directions to say they can use an if else or ternary